src/planscape/forsys/assert_dict_almost_equal.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+# Validates that two dictionaries have values that are "almost equal".
```

Comment:
Recommend adding a reference to https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_almost_equal.html or noting it's defaulting to numpy.aaae

Reply:
Done - I'ave added a comment.